### ISSUE TITLE
test_htmlizer fails on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8913]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8913|
|Type|enhancement|
|Created|2016-11-20 04:30:15Z|

https://github.com/twisted/twisted/pull/600

<details><summary>Searchable metadata</summary>

```
trac-id__8913 8913
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1479616215479488 1479616215479488
changetime__1480617823933097 1480617823933097
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph set owner to @rodrigc|
|-|-|

Review on the github ticket. Thanks for fixing this!

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set status to `closed`|
|-|-|

In changeset 91ff8029666decf9367bc72de12e21c53dc26736
```
#!CommitTicketReference repository="" revision="91ff8029666decf9367bc72de12e21c53dc26736"
Merge pull request #600 from twisted/8913-htmlizer-rodrigc

Author: rodrigc
Reviewer: glyph
Fixes: #8913

Port pyhtmlizer to Python 3
```
